### PR TITLE
TomTom as default LDS provider and reference documentation for cdb_bulk_geocode_street_point

### PR DESCRIPTION
**Pull request by danicarrion**

### CODE REVIEW

File: docs/developer-center/reference/05-geocoding-functions.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ INSERT INTO {tablename} (the_geom) SELECT cdb_geocode_ipaddress_point('102.23.34
 
 ### Street-Level Geocoder
 
-This function geocodes your data into a point geometry for a street address. CARTO platform uses [Mapbox geocoding services](https://www.mapbox.com/) by default as the service provider for street-level geocoding. [Contact us](mailto:sales@carto.com) if you have any specific questions or requirements about the location data service provider being used with your account.
+These functions geocode your data into a point geometry for a street address. CARTO platform uses [Mapbox geocoding services](https://www.mapbox.com/) by default as the service provider for street-level geocoding. [Contact us](mailto:sales@carto.com) if you have any specific questions or requirements about the location data service provider being used with your account.
```
Comment (javitonino) [inline docs/developer-center/reference/05-geocoding-functions.md]: We no longer use Mapbox as default (it's not TomTom). Maybe take the change to update it?

Comment (danicarrion) [reply docs/developer-center/reference/05-geocoding-functions.md]: I was planning to use a separate PR, as it affects more files and not to mix things. But I can also do it here, no strong preference.

Comment (javitonino) [reply docs/developer-center/reference/05-geocoding-functions.md]: Sure, as you prefer. Then :+1: to this PR

Comment (danicarrion) [reply docs/developer-center/reference/05-geocoding-functions.md]: it's just 2 files so I've added the changes here

**Pull request merged**